src/components/image/index.tsx

```diff
@@ -27,15 +50,22 @@ function Image({
     )
   }
 
+  /**
+   * We're passing width and height as style here to ensure they override
+   * any base styles.
+   */
+  const style = makeStyleProp(width, height)
```

Comment:
💭 Is `make` a common naming pattern we use in other places? I'm used to `generate`, but I like `make` quite a bit here. Or maybe, are `generate` and `make` actually different?? It's probably not be that deep, just a curious question. 😊

Reply:
Just did a quick search, and it looks like `generate` is used to name functions that return strings, objects, arrays, etc. that are built by the helper function (e.g. `generateStaticProps`, `generateStaticPaths`, `generatePackageManagers`, etc.). Seems like `make` is used to name functions that return a function/function component (e.g. `makeHeadingElement`, `makeFetchWithRetry`).

What do you think about using `make` and `generate` separately in this way? I formed my thoughts on this while writing this comment, so not deeply invested in the idea. If we come to a consensus, I think it'd be valuable to write it down (maybe in the `docs` folder). 😊

Reply:
Works for me! Thanks for doing some research here 🙏